src/pineko/evolve.py

```diff
@@ -105,6 +105,7 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
 
     """
     # Add a +1 to the orders for the difference in convention between nnpdf and pineappl
+    # NB: This would not happen for nFONLL
     is_fns = int(check.is_fonll_b(tcard["FNS"], pineappl_grid.lumi()))
```

Comment:
is the `tcard["PTO"]` for nFONLL already corrected? i.e. are we correcting here too much? (which can be an issue for N3LO) - so do you need to have a `is_num_fonll` in the condition?

Reply:
Indeed they are already corrected and in fact this is the meaning of the comment. The check `check.is_fonll_b` would return `False` for a nFONLL fk

Reply:
But shouldn't that check include `FONLL-D` as well ?

Reply:
In principle you are right and I can even add if you like, but we never use the `mixed FONLL` prescriptions beyond NLO

Reply:
[1b32ca0](https://github.com/NNPDF/pineko/pull/99/commits/1b32ca0ac3478b2a31e874bbfc5d6b967051162b)